server/src/routes/api/v1/servicesReviewsRouter.js

```diff
@@ -23,4 +23,16 @@ servicesReviewsRouter.post("/", async (req,res) =>{
   }
 })
 
+servicesReviewsRouter.delete('/', async (req, res) => {
+  try {
+    const { reviewId } = req.body
+    const reviewToDelete = await Review.query().findById(reviewId)
+    await reviewToDelete.$relatedQuery('votes').delete()
```

Comment:
We had to delete the votes associated with the review in order to delete the review.  Votes on the "votes" table require a review to exist, so we received an error if we attempted to delete a review with more than 0 total votes.  Is there a better way to delete a row on one table and it's dependent rows from other tables?  Will the same issue happen if we add a feature to delete services?  (ie.  deleting service requires deleting dependent reviews then deleting dependent votes).

Reply:
This is good! We want this level of annoyance in our application. 

The only thing you may want to know in the future is that for a manyToMany relation, you may need to also use `unrelate`